backend/app/processing/kb.py

```
@@ -0,0 +1,296 @@
+        faiss.write_index(index, index_path)
+        with open(docs_path, "w") as f:
+            json.dump(all_documents, f)
+        with open(texts_path, "w") as f:
+            json.dump(all_document_texts, f)
+        
+        # Update metadata
+        metadata = {
+            "indexed_files": file_metadata,
+            "document_count": len(all_documents),
+            "text_chunks_count": len(all_document_texts),
+            "last_updated": str(datetime.datetime.now())
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)